concordia/static/js/password-validation.js

```diff
@@ -0,0 +1,73 @@
+/* global jQuery */
+
+(function($) {
+    var requirements = [
+        {
+            id: 'pw-length',
+            text: 'At least 8 characters long',
+            test: function(i) {
+                return i.match(/.{8,}/);
+            }
+        },
+        {
+            id: 'pw-uppercase',
+            text: '1 or more uppercase characters',
+            test: function(i) {
+                return i.match(/[A-Z]/);
+            }
+        },
+        {
+            id: 'pw-digits',
+            text: '1 or more digits',
+            test: function(i) {
+                return i.match(/[0-9]/);
+            }
+        },
+        {
+            id: 'pw-special',
+            text: '1 or more non-unique special characters',
```

Comment:
Do we really have to say "non-unique" here, if we're allowing a single special character to satisfy the requirement? Are the digits non-unique as well?